test/liquid_handle_dispense_test.py

```diff
@@ -856,6 +861,170 @@ def test_tempest_bad_chip_param(self):
                 nozzle="abc",
             )
 
+    def test_mantis_default_params(self):
+        """Tests mantis default params"""
```

Comment:
are these mantis default params or are they device-agnostic params? like if we dont have a `mode_params` dict, are they even device specific?

Reply:
these are mantis default params! If we don't have a `mode_params` dict, it errors out!